Assets/Scripts/Managers/Quests.cs

```diff
@@ -39,10 +39,10 @@ private void Start()
         
         private void Update()
         {
-            if (!Input.GetMouseButtonDown(0) || !_selectedQuest) return;
-            if (_selectedQuest.mouseOver) return;
-            _selectedQuest.ResetDisplay();
-            _selectedQuest = null;
+            //if (!Input.GetMouseButtonDown(0) || !_selectedQuest) return;
+            //if (_selectedQuest.mouseOver) return;
+            //_selectedQuest.ResetDisplay();
+            //_selectedQuest = null;
```

Comment:
same deal here. delete old code if its not used anymore. don't leave a graveyard of comment